Test/Integration/AddScriptTest.php

```diff
@@ -14,52 +14,64 @@ class AddScriptTest extends AbstractController
      */
     private $script;
 
-    protected function setUp()
+    protected function setUp(): void
     {
         parent::setUp();
         $this->script = $this->_objectManager->create(ScriptGenerator::class)->generate();
     }
 
     /**
-     * @magentoConfigFixture default/web/cookiebot/enabled 1
-     * @magentoConfigFixture default/web/cookiebot/id 123-456-789
+     * @magentoConfigFixture current_store web/cookiebot/enabled 1
+     * @magentoConfigFixture current_store cookiebot/id 123-456-789
+     * @magentoConfigFixture current_store web/cookiebot/use_eu_cdn 1
      */
     public function testScriptAddedOnHomepage(): void
     {
         $this->dispatch('/');
-        self::assertContains($this->script, $this->getResponse()->getBody());
+        self::assertStringContainsString($this->script, $this->getResponse()->getBody());
+        self::assertEquals('<script id="Cookiebot" data-cfasync="false" src="https://consent.cookiebot.eu/uc.js" data-cbid=""  type="text/javascript" async></script>',
+            $this->script);
     }
 
     /**
-     * @magentoDataFixture   Magento/Catalog/_files/product_simple.php
-     * @magentoConfigFixture default/web/cookiebot/enabled 1
-     * @magentoConfigFixture default/web/cookiebot/id 123-456-789
+     * @magentoApiDataFixture   Magento/Catalog/_files/product_simple.php
```

Comment:
```suggestion
     * @magentoDataFixture   Magento/Catalog/_files/product_simple.php
```

I guess this should be `magentoDataFixture`? Why with Api?

Reply:
because it just work with API, it got fail when it was without api

Reply:
Looks like it worked with `magentoDataFixture` before, and it still works with it for me. Also, reading through https://developer.adobe.com/commerce/webapi/get-started/web-api-functional-testing/#custom-annotations-for-data-fixtures, it doesn't sound like we should use `magentoApiDataFixture`.